packages/pluggableWidgets/floating-action-button-native/src/assets/FloatingActionButtonPrimary.ts

```diff
@@ -0,0 +1,15 @@
+export const FloatingActionButtonPrimarySVG = () =>
```

Comment:
Is there a specific reason to go for a function here? I think you can just assign the string literal to the constant here.

Reply:
You're right, updated!